docs/code-snippets/outlook-snippets.yaml

```diff
@@ -586,6 +586,8 @@ Office.AppointmentRead#itemId:member:
     // identifier. If the `itemId` property returns `null` or
     // `undefined`, it saves the item to the store and gets the
     // item identifier from the asynchronous result.
+    // Important: saveAsync was introduced with requirement set 1.3
```

Comment:
There's a bit of inconsistency with how code ticks are used in this comment. `itemId` should either always be surrounded or never.